builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -244,7 +244,13 @@ func resourceAwsAutoscalingGroup() *schema.Resource {
 				},
 			},
 
-			"tag": autoscalingTagsSchema(),
+			"tag": autoscalingTagSchema(),
+
+			"tags": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeMap},
+			},
```

Comment:
Is there any reason we should accept both fields and deal with all kinds of potential edge cases related to that? If not, would you mind making them mutually exclusive via `ConflictsWith`?